app/views/home/landing.html.erb

```diff
@@ -45,10 +45,10 @@
         </div>
       </div>
       <section class="queryOptionsContainer" id="default">
-        <% @capabilities.to_a.sample(5).each do |c| %>
+        <% @capabilities.to_a.each do |c| %>
```

Comment:
If we ended up having too many capabilities, would showing all of them be too much??

Reply:
This was a petition made by the client.

Reply:
I can confirm this is something the product owner asked for specifically, despite our recommendation to limit the amount.